### PR TITLE
user: create token cli

### PR DESCRIPTION
**Pull request by rerowep**
* Deletes obsolete code for user token creation.
* Replaces all string format functions with f strings.

### CODE REVIEW

File: rero_ils/modules/imports/serializers/serializers.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def sort_ordered_dict(ordered_dict):
             for key, value in ordered_dict.items():
                 if key != '__order__':
                     if len(key) == 5:
-                        key = '{tag} {ind}'.format(tag=key[:3], ind=key[3:])
+                        key = '{key[:3]} {key[3:]}'
```
Comment (jma) [inline rero_ils/modules/imports/serializers/serializers.py:164]: `f` seems missing.

File: scripts/setup [hunk 2]
```diff
@@ -76,6 +76,16 @@ invert_warning_option() {
   fi
 }
 
+token_create() {
+  if [ -z ${3} ]
+  echo "eval ${PREFIX} invenio reroils utils token-create -n ${1} -u ${2} -t ${3}"
```
Comment (jma) [inline scripts/setup:81]: Does it makes sense? You can use a message instead.

File: scripts/setup [hunk 3]
```diff
@@ -76,6 +76,16 @@ invert_warning_option() {
   fi
 }
 
+token_create() {
```
Comment (jma) [inline scripts/setup:79]: create_token?

File: tests/api/documents/test_documents_rest.py [hunk 4]
```diff
@@ -61,7 +61,7 @@ def datetime_milliseconds(date):
     acq_future_timestamp = datetime_milliseconds(datetime_delta(days=1))
     today = today.strftime('%Y-%m-%d')
 
-    # # Add a new items with acq_date
+    # # Add a new item with acq_date
```
Comment (jma) [inline tests/api/documents/test_documents_rest.py:64]: You can remove one `#`

File: tests/api/item_types/test_item_types_rest.py [hunk 5]
```diff
@@ -125,7 +125,7 @@ def test_item_types_post_put_delete(client, org_martigny,
         headers=json_header
     )
     assert res.status_code == 200
-    # assert res.headers['ETag'] != '"{}"'.format(librarie.revision_id)
+    # assert res.headers['ETag'] != f'"{librarie.revision_id}"'
```
Comment (jma) [inline tests/api/item_types/test_item_types_rest.py:128]: To remove?

File: tests/api/items/test_items_rest.py [hunk 6]
```diff
@@ -193,7 +192,7 @@ def test_items_post_put_delete(client, document, loc_public_martigny,
         headers=json_header
     )
     assert res.status_code == 200
-    # assert res.headers['ETag'] != '"{}"'.format(librarie.revision_id)
+    # assert res.headers['ETag'] != f'"{librarie.revision_id}"'
```
Comment (jma) [inline tests/api/items/test_items_rest.py:195]: To remove?

File: tests/api/libraries/test_libraries_rest.py [hunk 7]
```diff
@@ -110,7 +110,7 @@ def test_libraries_post_put_delete(client, lib_martigny_data, json_header):
         headers=json_header
     )
     assert res.status_code == 200
-    # assert res.headers['ETag'] != '"{}"'.format(librarie.revision_id)
+    # assert res.headers['ETag'] != f'"{librarie.revision_id}"'
```
Comment (jma) [inline tests/api/libraries/test_libraries_rest.py:113]: Same

**Review by jma — CHANGES_REQUESTED** [review]

File: rero_ils/modules/imports/serializers/serializers.py (hunk 1 above)
Comment (rerowep) [reply rero_ils/modules/imports/serializers/serializers.py:164]: Well seen!

File: tests/api/item_types/test_item_types_rest.py (hunk 5 above)
Comment (rerowep) [reply tests/api/item_types/test_item_types_rest.py:128]: This exists in multiple places, so I have left it alone.

**Review by jma — APPROVED** [review]

File: rero_ils/modules/selfcheck/utils.py [hunk 8]
```diff
@@ -80,18 +80,16 @@ def format_patron_address(patron):
     """
     address = patron.get('second_address')
     if address:
-        formated_address = '{street}, {postal_code} {city}'.format(
-            street=address.get('street'),
-            postal_code=address.get('postal_code'),
-            city=address.get('city')
-        )
+        street = address.get('street'),
+        postal_code = address.get('postal_code'),
```
Comment (lauren-d) [inline rero_ils/modules/selfcheck/utils.py:84]: You forgot to remove comma at the end of line

File: rero_ils/modules/selfcheck/utils.py [hunk 9]
```diff
@@ -80,18 +80,16 @@ def format_patron_address(patron):
...
+        city = address.get('city')
+        formated_address = f'{street}, {postal_code} {city}'
     else:
         profile = patron.user.user_profile
-        formated_address = '{street}, {postal_code} {city}'.format(
-            street=profile['street'].strip(),
-            postal_code=profile['postal_code'].strip(),
-            city=profile['city'].strip()
-        )
+        street = profile['street'].strip(),
```
Comment (lauren-d) [inline rero_ils/modules/selfcheck/utils.py:89]: Same here. there is a comma at the end line

**Review by lauren-d — CHANGES_REQUESTED** [review]

**Review by lauren-d — APPROVED** [review]

**Pull request merged**